carma-messenger-core/cpp_message/CMakeLists.txt

```diff
@@ -16,8 +16,8 @@
 cmake_minimum_required(VERSION 2.8.3)
 project(cpp_message)
 
-## Compile as C++11, supported in ROS Kinetic and newer
-add_compile_options(-std=c++17)
+## Compile as C++14, supported in ROS Noetic and newer
+add_compile_options(-std=c++14)
```

Comment:
Looks like this was marked for c++17. I have to imagine there was a reason for this as its different from nearly all our other code. Was this able to build successfully?

Reply:
@msmcconnell  The change to C++ 17 came in with a commit (a56e3408a9e3d899aebf2afb292f26a395034333), "add unit tests." I don't see any C++ 17 specifics with this change. I also verified that all of the cpp files in cpp_message compile with -std=c++14 as well as -std=c++11.